src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -180,7 +180,7 @@ function bindEvents(elem) {
                 // to the previous track, unless we are at the first track so no previous track exists.
                 // currentTime is in msec.
 
-                if (playbackManager.currentTime(currentPlayer) >= 5 || playbackManager.getCurrentPlaylistIndex(currentPlayer) <= 1) {
+                if (playbackManager.currentTime(currentPlayer) >= 5 * 1000 || playbackManager.getCurrentPlaylistIndex(currentPlayer) <= 0) {
```

Comment:
I just prefer less computation.
```suggestion
                if (playbackManager.currentTime(currentPlayer) >= 5000 || playbackManager.getCurrentPlaylistIndex(currentPlayer) <= 0) {
```
Or maybe introduce a const.

Reply:
It will be optimized during build stage when building for production:
<img width="281" alt="image" src="https://github.com/jellyfin/jellyfin-web/assets/21142851/83157733-fc6d-4ab4-a203-6d894a320266">
I think it improves readability a bit, but I can change it if you don't think so.

Reply:
> It will be optimized during build stage when building for production:
> <img alt="image" width="281" src="https://private-user-images.githubusercontent.com/21142851/298381709-83157733-fc6d-4ab4-a203-6d894a320266.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU4NDA3ODEsIm5iZiI6MTcwNTg0MDQ4MSwicGF0aCI6Ii8yMTE0Mjg1MS8yOTgzODE3MDktODMxNTc3MzMtZmM2ZC00YWI0LWEyMDMtNmQ4OTRhMzIwMjY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIxVDEyMzQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQzMDdkNjIzZmJhYzliZjkyNjI5YjljNWM5YmViN2QzNzBmYWRmZDU5MGJhNTM1OGYwZWU1MjkzOWJhNGU5MGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ss3FPNQxw8w6UuxoABLX_IJpmcSwB9b5dIVXHKoBWs0">

OK then.